packages/element/src/serialize.js

```diff
@@ -36,6 +36,7 @@ import {
 	kebabCase,
 	isPlainObject,
 } from 'lodash';
+import CoreJSMap from 'core-js-pure/features/map';
```

Comment:
Any reason why we use this?

Reply:
It's meant to be temporary, an experiment to see if it would fix the previously-observed issue to use a very predictable implementation.